doc/source/rllib-algorithms.rst

```diff
@@ -1,6 +1,35 @@
 RLlib Algorithms
 ================
 
+.. tip::
+
+    Check out the `environments <rllib-env.html>`__ page to learn more about different environment types.
+
+Feature Compatibility Matrix
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+=============  =======================  ==================  ===========  ===========================
+Algorithm      Discrete Actions         Continuous          Multi-Agent  Model Support
```

Comment:
"Continuous" -> "Continuous Actions"

Reply:
The table gets too wide then unfortunately